clj-refactor.el

```diff
@@ -2689,9 +2689,9 @@ str/split => split"
   (let ((name (cljr--normalize-symbol-name symbol)))
     (cond
      ((string-match-p "\\w+\\.\\w+" name)
-      (thread-last name (s-split "\\.") last car cljr--symbol-suffix))
+      (thread-last name (split-string "\\.") last car cljr--symbol-suffix))
```

Comment:
I'd happily add the test for this, but seeing as `cljr--symbol-suffix` is a non-interactive function and most of the tests are feature tests, should I have a step like this:

```feature
When I press "M-: (cljr--symbol-suffix \"java.util.Date\")"
Then I should see message "Date"
```

What are your thoughts?